vhdbuilder/packer/post-install-dependencies.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+OS=$(sort -r /etc/*-release | gawk 'match($0, /^(ID_LIKE=(coreos)|ID=(.*))$/, a) { print toupper(a[2] a[3]); exit }')
+UBUNTU_OS_NAME="UBUNTU"
+
+source /home/packer/provision_installs.sh
+source /home/packer/provision_installs_distro.sh
+source /home/packer/provision_source.sh
+source /home/packer/provision_source_distro.sh
+source /home/packer/tool_installs.sh
+source /home/packer/tool_installs_distro.sh
+
+CPU_ARCH=$(getCPUArch)  #amd64 or arm64
+VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
+
+if [[ $OS == $MARINER_OS_NAME ]]; then
+  if grep -q "kata" <<< "$FEATURE_FLAGS"; then
+    enableMarinerKata
```

Comment:
shouldn't we do this before rebooting, since it generates a new grub config iirc?

Reply:
pretty sure we can at least swap this one sku over to standard_d4ads_v5 without issue, and we could probably swap everything (I think default quota is 100 cores, need to double check tho)

Reply:
should be able to test easily

Reply:
looks like this works, will use this approach for now and can hopefully upgrade the vm sizes for other vhd builds in the future as well